xds/src/main/java/io/grpc/xds/internal/sds/ServerWrapperForXds.java

```diff
@@ -48,6 +50,12 @@ public Server start() throws IOException {
     return this;
   }
 
+  @VisibleForTesting
+  public Server startForTestXdsClient() throws IOException {
```

Comment:
why we need this? test seems fine with using the start

Reply:
This is needed for tests in `XdsSdsClientServerTest`. To avoid an actual XDS server, I use a mock XDS client that I inject using `XdsClientWrapperForServerSds.start(XdsClient xdsClient)`. So when a regular `ServerWrapperForXds.start` is called I get the "start() called more than once" exception. Under the circumstances I thought this was the best solution.

Reply:
The other option is to remove the check for "start() called more than once" and instead silently return from `XdsClientWrapperForServerSds.start()` if xdsClient is set. Since end users never use `XdsClientWrapperForServerSds` directly this may not be too bad.